### PR TITLE
Prepare for 3.4.0 release

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+* `resource/nsxt_policy_security_policy`, `resource/nsxt_policy_gateway_policy`: Validate correctness of sequence numbers only on policy creation, and skip this check on update, but rather auto-correct sequence numbers if needed. This is in order to avoid erroring out in case of incorrect sequence numbers that got assigned in previous provider version ([#1001](https://github.com/vmware/terraform-provider-nsxt/pull/1001))
+* Escape all special characters in data sources, as required by search API. This fixes and issue with search by `display_name` that was not working as expected in case it contained some special characters ([#993](https://github.com/vmware/terraform-provider-nsxt/pull/993))
+
+EXPERIMENTAL FEATURES:
+Fabric support is offered as Beta with this release:
+
+* `data/nsxt_compute_collection`
+* `data/nsxt_compute_manager`
+* `data/nsxt_failure_domain`
+* `data/nsxt_uplink_host_switch_profile`
```
Comment (ksamoray) [inline CHANGELOG.md:21]: nsxt_policy_uplink_host_switch_profile

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+
+* `resource/nsxt_cluster_virtual_ip`
+* `resource/nsxt_compute_manager`
+* `resource/nsxt_edge_cluster`
+* `resource/nsxt_failure_domain`
+* `resource/nsxt_manager_cluster`
+* `resource/nsxt_policy_host_transport_node_profile`
+* `resource/nsxt_policy_transport_zone`
+* `resource/nsxt_transport_node`
+* `resource/nsxt_uplink_host_switch_profile`
```
Comment (ksamoray) [inline CHANGELOG.md:33]: resource/nsxt_policy_uplink_host_switch_profile

**Review by ksamoray — COMMENTED** [review]
I've diffed provider.go, the following are missing, IDK if you left those out on purpose:

resource/nsxt_policy_transport_zone
resource/nsxt_policy_user_management_role
resource/nsxt_policy_user_management_role_binding
resource/nsxt_node_user
resource/nsxt_policy_host_transport_node
resource/nsxt_edge_high_availability_profile
resource/nsxt_policy_host_transport_node_collection

**Comment by salv-orlando** [comment]
Besides Kobi comments, LGTM

**Review by ksamoray — APPROVED** [review]

**Pull request merged**